changelogs/unreleased/gh-6200-lua_cjson-encode-escapes-slash.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/lua-cjson
```

Comment:
`bugfix/lua` or `bugfix/lua/json`. Our built-in module is called just `json` from a user's perspective.

Reply:
Done